docs/components/connectors/react-components/_connectors-table.js

```diff
@@ -0,0 +1,527 @@
+      link: "../slack/?slack=outbound",
+      image: SlackImg,
+    },
+    {
+      name: "SOAP",
+      description:
+        "Connect with Simple Object Access Protocol (SOAP) services.",
+      type: "Outbound",
+      link: "../../protocol/soap/",
+      image: SoapImg,
+    },
+    {
+      name: "SQL",
+      description:
+        "Connect with SQL databases (Microsoft SQL Server, PostgreSQL, MySQL).",
+      type: "Outbound",
+      link: "../sql/",
+      image: SqlImg,
+    },
+    {
+      name: "Twilio",
+      description:
+        "Send and get SMS messages with Twilio service from your processes.",
+      type: "Outbound",
+      link: "../twilio/?twilio=outbound",
+      image: TwilioImg,
+    },
+    {
+      name: "Twilio Webhook",
+      description: "Start a process instance triggered by a Twilio webhook.",
+      type: "Inbound",
+      link: "../twilio/?twilio=inbound",
+      image: TwilioImg,
+    },
+    {
+      name: "UiPath",
+      description: "Orchestrate your UiPath Bots with Camunda.",
+      type: "Outbound",
+      link: "../uipath/",
+      image: UipathImg,
+    },
+    {
+      name: "WhatsApp",
+      description: "Send messages with WhatsApp Business from your processes.",
+      type: "Outbound",
+      link: "../whatsapp/",
+      image: WhatsappImg,
+    },
+  ];
+
+  const handleSearch = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
+  const handleFilterType = (type) => {
+    setFilterType(type);
+  };
+
+  const filteredConnectors = connectors.filter((connector) => {
+    const matchesSearchTerm =
+      connector.name.toLowerCase().includes(searchTerm.toLowerCase()) ||
+      connector.description.toLowerCase().includes(searchTerm.toLowerCase());
+    const matchesFilterType =
+      filterType === "All" || connector.type === filterType;
+    return matchesSearchTerm && matchesFilterType;
+  });
+
+  return (
+    <div className="connector-input">
+      <input
+        type="text"
+        placeholder="Find a Camunda connector"
+        value={searchTerm}
+        onChange={handleSearch}
+        className="connector-input-box"
+        style={{ marginRight: "20px" }}
+      />
+      <div className="filter-badges">
+        <span
+          className={`badge--default ${
+            filterType === "All" ? "badge--active" : ""
+          }`}
+          onClick={() => handleFilterType("All")}
+        >
+          All
+        </span>
+        <span
+          className={`badge--default ${
+            filterType === "Inbound" ? "badge--active--inbound" : ""
+          }`}
+          onClick={() => handleFilterType("Inbound")}
+        >
+          Inbound
+        </span>
+        <span
+          className={`badge--default ${
+            filterType === "Outbound" ? "badge--active--outbound" : ""
+          }`}
+          onClick={() => handleFilterType("Outbound")}
+        >
+          Outbound
+        </span>
+      </div>
+      <span style={{ fontSize: "14px" }}>
```

Comment:
non-blocking: Any particular reason for inline styling here but CSS classes elsewhere?